handbook/company/pricing-features-table.yml

```diff
@@ -697,7 +681,8 @@
     - description: Install a bootstrap package to run custom scripts during the setup experience. Store the bootstrap package outside the Fleet database coming soon (2024-09-15) #customer-faltona
       moreInfoUrl: https://github.com/fleetdm/fleet/issues/19037
     - description: Require end users to authenticate with your identity provider (IdP) and agree to an end user license agreement (EULA) before they can use their new workstation
-- industryName: Enforce OS updates
+- industryName: Enforce operating system (OS) updates
```

Comment:
Spelled out the acronym (this will make more sense after the re-ordering since it will be listed first) and updated the description/tooltip.